cypress/support/commands.ts

```diff
@@ -87,33 +87,38 @@ export const loginViaDex = (username: string, password: string) => {
           followRedirect: false,
           method: 'GET',
         })
-        .then(secondResp => {
-          cy.request({
-            url: Cypress.env(DEX_URL_VAR) + secondResp.headers.location,
-            method: 'POST',
-            form: true,
-            body: {
-              login: username,
-              password: password,
-            },
-            followRedirect: false,
-          }).then(thirdResp => {
-            const req = (thirdResp.headers.location as string).split(
-              '/approval?req='
-            )[1]
-            cy.request({
-              url: thirdResp.redirectedToUrl,
-              followRedirect: true,
-              form: true,
+        .then(secondResp =>
+          cy
+            .request({
+              url: Cypress.env(DEX_URL_VAR) + secondResp.headers.location,
               method: 'POST',
-              body: {req: req, approval: 'approve'},
-            }).then(() => {
-              cy.visit('/')
-              // cy.getCookie('session').should('exist')
-              cy.location('pathname').should('not.eq', '/signin')
+              form: true,
+              body: {
+                login: username,
+                password: password,
+              },
+              followRedirect: false,
             })
-          })
-        })
+            .then(thirdResp => {
+              const req = (thirdResp.headers.location as string).split(
+                '/approval?req='
+              )[1]
+              return cy
+                .request({
+                  url: thirdResp.redirectedToUrl,
+                  followRedirect: true,
+                  form: true,
+                  method: 'POST',
+                  body: {req: req, approval: 'approve'},
+                })
+                .then(() => {
+                  cy.visit('/')
+                  // cy.getCookie('session').should('exist')
+                  cy.location('pathname').should('not.eq', '/signin')
+                  return cy.getByTestID('tree-nav')
```

Comment:
this diff is all sorts of wacky. This line is the main change, right?

Reply:
yeah I also cleaned up the returns - changed ` => {` to `=>`